Move Galaxy showcase out of sandbox

wm75 opened the pull request:
@rabuono sorry for the delay, but your review is very welcome now.

rabuono commented:
Closes #167

rabuono reviewed (commented):
I managed a first pass
Can you please check the suggestions and commit what you agree with?
Then I can give it another go.
We should add a few items to the "tools and resources table".
Also might be worth moving the `how to reuse the components` to inside of `what can you use this showcase for`

wm75 commented on showcase/covid19-galaxy.md:
```diff
@@ -0,0 +1,129 @@
+Fortunately, investments made in public and open computing and storage infrastructure projects, as well as in reliable and scalable data analysis
+frameworks have proven beneficial and could be employed for the required genome monitoring of SARS-CoV-2.
+
+Here, we demonstrate how the [Galaxy Project](https://galaxyproject.org/projects/covid19) with the help of many additional collaborators leveraged
```
> and keeping that full link for the link of the showcase itself.

What would that look like?

rabuono replied:
```suggestion
Here, we describe a modular and scalable [system for FAIR analysis of SARS-CoV-2 sequencing data](https://galaxyproject.org/projects/covid19/). This effort shows how the [Galaxy Project](https://galaxyproject.org/) with the help of many additional collaborators repurposed, expanded, and combined public infrastructure and systems for storing and sharing data, and for managing data analysis workflows, to tackle a global health emergency.
system for FAIR analysis of SARS-CoV-2 sequencing data.
```

rabuono replied:
I added a suggestion.
Would give a more direct link to the showcase, wile still making the point of the overall Galaxy Project.

wm75 commented:
> Might be nice to add here a link not only the query for COVID under the iwc Team in WorfklowHub. That is nice and should be kept. But in addition to it, it would be nice to have a WorkflowHub Collection that contains the workflows used during the original response. Part of the effort of the showcase is to have an example of what was done. Then add a link here to said collection.

@rabuono I'm not sure about this. A collection like you imagine even exists: https://workflowhub.eu/collections/2 (so was even the second collection ever created :).
However, it lacks two WFs I would like to have visible and I'm afraid it would be really confusing for readers to have two links that lead to slightly different sets of WFs.

wm75 commented:
Apart from the workflowhub collection idea and the open question about the link to the Galaxy project, I think I've addressed all your review comments, @rabuono.

rabuono commented:
> > Might be nice to add here a link not only the query for COVID under the iwc Team in WorfklowHub. That is nice and should be kept. But in addition to it, it would be nice to have a WorkflowHub Collection that contains the workflows used during the original response. Part of the effort of the showcase is to have an example of what was done. Then add a link here to said collection.
> 
> @rabuono I'm not sure about this. A collection like you imagine even exists: https://workflowhub.eu/collections/2 (so was even the second collection ever created :). However, it lacks two WFs I would like to have visible and I'm afraid it would be really confusing for readers to have two links that lead to slightly different sets of WFs.

I see your point. I added a minor change to try and make more clear that the workflows are an evolving list. Since the link is to a query result, the workflows that are going to be found will depend on which workflows are present with those tags at the time of the visit to WorkflowHub

rabuono commented on showcase/covid19-galaxy.md:
```diff
@@ -0,0 +1,149 @@
+contributors: [Wolfgang Maier] 
+description: An automated, modular system for large-scale FAIR analysis of SARS-CoV-2 sequencing data analysis powered by the Galaxy platform.
+affiliations: [DE, European Union]
+page_id: covid19-galaxy-project
```
```suggestion
page_id: covid19_galaxy_project
```

rabuono commented on showcase/covid19-galaxy.md:
```diff
@@ -0,0 +1,149 @@
+
+### Viral Beacon project
+
+In the showcase the [Viral Beacon project](https://covid19beacon.crg.eu/) acts as a consumer of the data and provides visualisations.
```
```suggestion
In the showcase the [ CRG Viral Beacon](https://covid19beacon.crg.eu/) acts as a consumer of the data and provides visualisations.
```
Added specific name of the tool to avoid confusion with the Beacon project in general

rabuono replied:
Also adding CRG Viral Beacon to the tools and resources table

rabuono commented on showcase/covid19-galaxy.md:
```diff
@@ -0,0 +1,150 @@
+frameworks have proven beneficial and could be employed for the required genome monitoring of SARS-CoV-2.
+
+Here, we describe a modular and scalable [system for FAIR analysis of SARS-CoV-2 sequencing data](https://galaxyproject.org/projects/covid19/). This effort shows how the [Galaxy Project](https://galaxyproject.org/) with the help of many additional collaborators repurposed, expanded, and combined public infrastructure and systems for storing and sharing data, and for managing data analysis workflo ...
+system for FAIR analysis of SARS-CoV-2 sequencing data.
+and combined public infrastructure and systems for storing and sharing data, and for managing data analysis workflows, to form a modular and scalable
+system for FAIR analysis of SARS-CoV-2 sequencing data.
```
```suggestion
```
repeated text
